src/components/progress-report/variance-explanation/variance-explanation.repository.ts

```diff
@@ -73,6 +73,6 @@ export class ProgressReportVarianceExplanationRepository extends DtoRepository(
       .return('*')
       .run();
 
-    return await this.readOne(id);
+    return undefined as unknown;
```

Comment:
I figured, all good. The important thing is to not do an additional readOne when the DB could just return it. I don't care too much about the neo4j repositories since they will be deleted soon™️. The patterns in services matter more. These progress report children services are more complicated too because trying to get at the project sensitivity soon deep in the tree. There's an opportunity to rewrite these after migration.